front_end/migration_scripts/17_to_18_check.sql

```diff
@@ -0,0 +1,3 @@
+SELECT COUNT(*) AS count
```

Comment:
I'm not sure if this will work for the next migration. Will it try to run this again and then throw an error?

If so, you should be able to query pragma_table_info and see how many columns are in the users table. If this indicates that the picture column is still there, then you could do the migration.

Reply:
Oh sorry, I forgot that I was going to ask you about this too! Correct me if I'm wrong, but we will have to edit the ```migrate.py``` script that runs each sql file no matter what, right? It checks the count in each file to see whether they are over 0, but in this case we want to either know the opposite or ensure that the number of columns in ```user``` is 7, rather than 8. If we do in fact need to change ```migrate.py```, would you rather I hard coded this particular check (17_to_18) into it or made some way (perhaps by indicating in the filename) that this check is to ensure there isn't a column present?

Reply:
Never mind, I spoke too soon! I think I understand what you were saying about pragma_table_info now, this code is working for me.
``` SELECT COUNT(*) AS count
	FROM (SELECT COUNT(*) AS col_count
			FROM pragma_table_info("users"))
				WHERE col_count = 7```